logic/src/commonMain/kotlin/com/wire/kalium/logic/data/notification/DbNotification.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.logic.data.notification
+
+import com.wire.kalium.logic.data.id.ConversationId
+
+data class DbNotificationConversation(
```

Comment:
I was wondering if these `Db` prefixes felt right, or maybe would create some kind of confusion, but I don't have an answer either 🤔  😅 

I was thinking in something like `Local` instead of `Db`, let me know if it makes sense to you =)